### PR TITLE
FS-1450 Add sign-out link

### PR DESCRIPTION
**Pull request by harryyo**
### Add sign-out link

- "Sign out" link added to header
- js menu drop down added which appears on small screens when js is enabled

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.18.tar.gz
+git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1450-is-authenticated-flag
```
Comment (srh-sloan) [inline requirements.in:4]: Needs to be a concrete version before merging

**Review by srh-sloan — APPROVED** [review]
Looks good, just need to update the fsd_utils version to a released one before merging please :)

File: requirements.in (hunk 1 above)
Comment (harryyo) [reply requirements.in:4]: Thanks, awaiting new version number on utils

**Review by gio-karunakaran — APPROVED** [review]
Looks great!

File: requirements.in (hunk 1 above)
Comment (harryyo) [reply requirements.in]: Updated to 0.0.21

**Pull request merged**